### PR TITLE
Automate Dev vs Production trackingId use

### LINKED ISSUE #614: Automate production analytics vs dev mode
We have 2 google analytics asset-ids so we can keep analytics on while developing, however this requires remembering to change the trackingId before an npm publish, which could be error prone.

### PR DESCRIPTION
**Pull request by purplecabbage**
if .git exists use the dev tracking code
This closes #614

### CODE REVIEW

**Comment by purplecabbage** [comment]
hold for more testing, and review.

File: lib/cli/analytics.js [hunk 1]
```diff
@@ -16,10 +16,21 @@ phonegap analytics reset|on|off
...
+    // set it to the dev tracking code
+    trackingCode = 'UA-94271-38';
+}
+
 var insight = new Insight({
-    // Google Analytics tracking code
-    // for release, use 'UA-94271-37'
-    "trackingCode": 'UA-94271-38',
+    "trackingCode": trackingCode,
     "pkg": require('../../package.json')
```
Comment (mwbrooks) [inline lib/cli/analytics.js]: single quotes on `'pkg'` ;)

**Comment by mwbrooks** [comment]
Looks good to me @purplecabbage. Checking for `.git` covers local development, locally linked copies, and automated test environments. So, I can't think of any cases where something might leak into production.

In the future, it would be nice to do development on and send pull requests from the phonegap repo instead of your own fork. It makes collaboration between the team easier.

File: lib/cli/analytics.js (hunk 1 above)
Comment (purplecabbage) [inline lib/cli/analytics.js]: Object literals should have double quotes on property names. This comes from the JSON spec.
Also, that is not a part of this pr.

**Pull request merged**